akka-stream-tests/src/test/java/akka/stream/javadsl/FlowTest.java

```diff
@@ -382,6 +383,55 @@ public void mustBeAbleToUseMerge() throws Exception {
   }
 
   @Test
+  public void mustBeAbleToUseAsyncMerge() throws Exception {
```

Comment:
I guess this is also the case for some other stages tested there (and already covered elsewhere)